### DISCUSSION TITLE
Nullable DateTimePicker

### DISCUSSION THREAD

**Discussion by Zeljko-Predjeskovic** [CommunityToolkit/Maui#1346] [New Feature Discussions]
I was recently implementing a nullable DateTimePicker in the dotnet maui project. But I do not think that they will merge my implementation into their project. 
Since I want to have that feature, I was thinking of implementing it in the community toolkit.

**Comment by bijington** [comment]
I know that I certainly had value for it in Xamarin.Forms projects so I can't see why it wouldn't be of value for .NET MAUI. Do you have more details on what you are proposing?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
